src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchServiceTests.cs

```diff
@@ -146,6 +146,11 @@ public TestSearchService(ISearchOptionsFactory searchOptionsFactory, IFhirDataSt
             {
                 return Task.FromResult(SearchImplementation(searchOptions));
             }
+
+            protected override Task<SearchResult> SearchForReindexInternalAsync(SearchOptions searchOptions, string searchParameterHash, CancellationToken cancellationToken)
```

Comment:
do we need to add any UTs for this new method?

Reply:
I added an additional unit test.